Lib/idlelib/colorizer.py

```diff
@@ -14,8 +15,22 @@ def any(name, alternates):
     return "(?P<%s>" % name + "|".join(alternates) + ")"
 
 
-def make_pat():
+def make_pats():
     kw = r"\b" + any("KEYWORD", keyword.kwlist) + r"\b"
+    match_softkw = (
+        r"^[ \t]*" +  # at beginning of line + possible indentation
+        r"(?P<MATCH_SOFTKW>match)\b" +
+        r"(?!(?:[ \t]|\\\n)*[=,)\]}])"  # not followed by any of =,)]}
+    )
```

Comment:
When another statement directly follows, I prefer to have the dedent at the end of the statement, instead of before.
```suggestion
        )
```
At the end of a function, I don't care so much.

Reply:
Done.